frontend/src/components/lease/List.tsx

```diff
@@ -0,0 +1,24 @@
+import { useTranslation } from 'react-i18next';
+import { Lease } from '../../lib/k8s/lease';
+import { SectionBox, SectionFilterHeader } from '../common';
+import ResourceTable from '../common/Resource/ResourceTable';
+
+export function LeaseList() {
+  const { t } = useTranslation(['glossary', 'leases']);
+  return (
+    <SectionBox title={<SectionFilterHeader title={t('glossary|Lease')} />}>
+      <ResourceTable
+        resourceClass={Lease}
+        columns={[
+          'name',
+          'namespace',
+          {
+            label: t('frequent|Holder'),
```

Comment:
You don't have frequent in the pre-load of useTranslation.

Reply:
updated